src/components/sbb-pearl-chain-vertical/sbb-pearl-chain-vertical.scss

```diff
@@ -2,5 +2,5 @@
 
 .sbb-pearl-chain-vertical {
   display: table;
-  table-layout: fixed;
+  width: var(--sbb-pearl-chain-vertical-width, 100%);
```

Comment:
also here, I'd suggest to assign  100% to --sbb-pearl-chain-vertical-width on host:() level